syncthing_gtk/editordialog.py

```diff
@@ -405,42 +410,42 @@ def store_value(self, key, w):
 
     def cb_format_value_s(self, spinner):
         """Formats spinner value"""
-        spinner.get_buffer().set_text(_("%ss") % (int(spinner.get_adjustment().get_value()),), -1)
+        spinner.get_delegate().set_text(_("%ss") % (int(spinner.get_adjustment().get_value()),))
         return True
 
     def cb_format_value_s_or_disabed(self, spinner):
```

Comment:
Looks like a typo, should probably get renamed. This is unrelated to the GTK4 port though.
```suggestion
    def cb_format_value_s_or_disabled(self, spinner):
```